spec/LandingANDcontactUs/LandingANDContactUs.txt

```diff
@@ -0,0 +1,16 @@
+1 - Contact Us
+
+Verify that user can open 'Contact us' page
+Verify that user can choose file in 'Attach File' field
+Verify that user can change 'Subject Heading' dropdown menu
+Verify that user can send a message with entered valid data
+Verify that user can't send a message without entered data
+
```

Comment:
What about negative checks in email field? non-existent, empty, invalid format, etc.?
What about clicking 'Send' button? What we expect upon it?
Message text field?
Browser button 'Back', 'Refresh'?
'Home' button?